### ISSUE TITLE
location.assign() fails to trigger RaiseChange() / never reloads

### ISSUE THREAD

**Issue by DruSatori** [AngleSharp/AngleSharp#496]
MWE:

```cs
var config = Configuration.Default.WithJavaScript().WithDefaultLoader();
var context = BrowsingContext.New(config);
await context.OpenAsync(res => res.Content(@"
<title>I am the homepage</title>
<script>window.location.assign('http://www.google.com');</script>
"));

for (var i = 0; i < 10; i++)
{
    Console.WriteLine(context.Active.Title);
    await Task.Delay(1000);
}
```

**Comment by FlorianRappl** [comment]
Thanks for the report 👍 ! Will be fixed in the upcoming version.

**Comment by FlorianRappl** [comment]
Fixed, see #497.
